### ISSUE TITLE
Did you ever actually finish this?

### ISSUE THREAD

**Issue by liamzee** [tssm/up-to-date-real-world-haskell#42]
On the readme, it sounds like several chapters are in the todo / started status. But I'm not sure if the readme is still current.

**Comment by ocramz** [comment]
Agree, I just skimmed the chapters and they are all done! 🎉 the readme could be updated.
